priv/repo/seeds.exs

```diff
@@ -133,3 +133,15 @@ CoursePlanner.Repo.delete_all CoursePlanner.User
       required: false
     })
 |> CoursePlanner.Repo.insert!
+
+%CoursePlanner.SystemVariable{}
+|> CoursePlanner.SystemVariable.changeset(
+    %{
+      key: "DISABLE_NOTIFICATION",
+      value: "true",
+      type: "boolean",
+      editable: true,
+      visible: true,
+      required: false
```

Comment:
that is alright. Required field controls the update action. it allows the user to keep it as nil.